volttron/utils/__init__.py

```diff
@@ -35,6 +35,7 @@
 # BATTELLE for the UNITED STATES DEPARTMENT OF ENERGY
 # under Contract DE-AC05-76RL01830
 # }}}
+"""Some important modules are imported here to allow for implicit imports of such modules."""
```

Comment:
? aren't all modules that are used important.

This is the doc string so it should be something descriptive of what the4 utils package has in it.

"""
The volttron.utils package contains generic utilities for handling json, storing configurations
math libraries...and more
"""